js/ui/grid_core/ui.grid_core.rows.js

```diff
@@ -923,7 +923,10 @@ export const rowsModule = {
 
                     if(dxScrollable) {
                         dxScrollable.update();
-                        this._updateHorizontalScrollPosition();
+                        // TODO: seems that it method doesn't need for renovated widget
```

Comment:
Can we add one new test for this 'doesn't need' and run it on both the current sources and renovated scrollview?